CMakeLists.txt

```
@@ -101,9 +110,8 @@ CONFIGURE_FILE(
 ## TESTS
 ## create and configure the unit test target
 ##
-include(CTest) #adds option BUILD_TESTING (default ON)
-
-if(BUILD_TESTING AND JSON_BuildTests)
+if (JSON_BuildTests)
+    include(CTest) #adds option BUILD_TESTING (default ON)
```

```suggestion
    include(CTest)
```

This comment is not needed any more.